catalyst/dl/runner/wandb.py

```diff
@@ -16,8 +16,19 @@ class WandbRunner(Runner):
     """
     @staticmethod
     def _log_metrics(metrics: Dict, mode: str, suffix: str = ""):
+        def key_locate(key: str):
+            """
+            wandb used first symbol _ for it service purposes,
+            because of that fact, we can not send original metric names
+            :param key: metric name
```

Comment:
Please convert the documentation to Google Style :)

Reply:
Done)